DEVELOPMENT.md

```diff
@@ -9,21 +9,47 @@ The Pixie project currently consists of three main components:
 This document outlines the process for setting up the development environment for each of these components.
 
 ## Setting up the Environment
+There are two different ways to run a development environment. The first and easiest
+approach is via Docker. If you plan to use a non-Minikube environment (GKE, EKS, etc)
+for running Vizier/Pixie Cloud, use this first approach via Docker. The second approach
+is using Make and Minikube.
 
-To setup the developer environment required to start building Pixie's components, it is easiest to use our docker image which has the necessary tools and packages. If you plan to use a non-minikube environment (gke, eks, etc) for running Vizier/Pixie Cloud, this can be done by running our script:
+### Running Via Docker
+To set up the developer environment required to start building Pixie's components,
+it is easiest to run the `run_docker.sh` script. The following script will run the
+Docker container and dump you out inside the docker container console from which
+you can run all the necessary tools to build, test, and deploy Pixie.
 
-```
-./scripts/run_docker.sh
-```
+1. Since this script runs a Docker container, you must have Docker installed. To
+install it follow these instructions [here](https://docs.docker.com/get-docker/).
 
-Otherwise, to launch the dev environment with minikube, run:
+1. `run_docker.sh` requires the realpath command which is part of the coreutils
+package that you may need to install:
+    * Ubuntu: `sudo apt-get install coreutils`
+    * OS X: `brew install coreutils`
 
-```
-make dev-env-start
-```
+1. Finally, run the following script to start the Docker container:
+    ```
+    ./scripts/run_docker.sh
+    ```
+
+### Running Via Minikube
+If you plan on using a Minikube environment, launch the dev environment with Minikube,
+via Make.
+
+1. Since this runs via Minikube, you must have Minikube installed. To install it follow
+the instructions [here](https://minikube.sigs.k8s.io/docs/start/).
+
+1. Run make to spin up a Minikube environment:
+    ```
+    make dev-env-start
+    ```
 
 
 ## Pixie Cloud
+Pixie Cloud manages users, authentication, and proxying “passthrough” mode. If you want
+to make changes to it then you will need to spin up a self-hosted version to test those
+changes rather than pointing to the official cloud hosted version.
```

Comment:
Let's say, "If you want to make changes to Pixie Cloud, then you will need to spin up a self-hosted version in development mode to test those changes. If you aren't changing Pixie Cloud, feel free to use officially released Pixie Cloud options listed in our [Install Guides](https://docs.px.dev/installing-pixie/install-guides/)."